### PR TITLE
ccxt update & bybitv3 & avoid telegram error spam & lib updates & fix tradingview quantity

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/776
requires https://github.com/Drakkar-Software/OctoBot-Services/pull/137
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/780

### CODE REVIEW

File: Trading/Exchange/bybit/bybit_exchange.py [hunk 1]
```diff
@@ -90,27 +94,11 @@ def get_default_type(self):
     async def get_positions(self) -> list:
         params = {}
         raw_positions = []
-        for position_type in ("linear", "inverse"):
-            params["subType"] = position_type
+        for settleCoin in ("USDT", "BTC"):
+            params["settleCoin"] = settleCoin
```
Comment (Herklos) [inline Trading/Exchange/bybit/bybit_exchange.py:98]: :open_mouth:

Comment (GuillaumeDSM) [reply Trading/Exchange/bybit/bybit_exchange.py:98]: bybit is spliting positions with settlement assets

**Review by Herklos — APPROVED** [review]
💯

**Review by Herklos — APPROVED** [review]
👍

**Review by Herklos — APPROVED** [review]

**Pull request merged**